tests/helpers/test_event.py

```diff
@@ -539,7 +539,7 @@ async def test_track_time_interval(hass):
 
     utc_now = dt_util.utcnow()
     unsub = async_track_time_interval(
-        hass, lambda x: specific_runs.append(1), timedelta(seconds=10)
+        hass, lambda x: specific_runs.append(x), timedelta(seconds=10)
```

Comment:
Lambdas will be scheduled on the executor. Not sure if that is intended.

Reply:
It will work, but its not ideal

Reply:
Fixed here https://github.com/home-assistant/core/pull/38212